src/main/resources/application.yaml

```
@@ -20,11 +20,35 @@ spring:
         hibernate:
             ddl-auto: create-drop
-        show-sql: true
+        show-sql: false
         properties:
             hibernate:
-                format_sql: true
+                format_sql: false
+
+# Logging Configuration
+logging:
+  level:
```

YAMLlint (1.37.1)

[warning] wrong indentation: expected 4 but found 2

(indentation)